c++/src/capnp/rust_capnp_library.bzl

```diff
@@ -0,0 +1,82 @@
+"""Bazel rule to compile .capnp files into rust."""
+
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
+load("@capnp-cpp//src/capnp:capnp_gen.bzl", "capnp_gen", _capnp_provider = "capnp_provider")
+load("@rules_rust//rust:defs.bzl", "rust_library")
+
+capnp_provider = _capnp_provider
+
+def rust_capnp_library(
+        name,
+        srcs,
+        crate_name,
+        data = [],
+        deps = [],
+        src_prefix = "",
+        tags = [],
+        visibility = None,
+        target_compatible_with = None,
```

Comment:
Nit: target_compatible_with and visibility can be omitted if you also add `**kwargs` to the `capnp_gen()` invocation.

Reply:
This is not exactly what I want: rust_library has lots of additional arguments that don't need to be passed to other targets here.